app/assets/stylesheets/components/_tax-return-list.scss

```diff
@@ -171,5 +171,32 @@
   line-height: $font-size-18;
   padding: $s5 $s10;
   border: 1px solid $color-green-money;
-}
 
+  // More info about styling with data attributes can be found at
+  // https://css-tricks.com/a-complete-guide-to-data-attributes/
+  &[data-status="Intake/Not ready"],
+  &[data-status="Admisión/No listo"],
```

Comment:
@bytheway875 added support for Spanish labels. I wonder now if this isn't the best approach, but I figured if the CSS follow the label content, it would be less logic to maintain. But now there's this CSS logic so 🤷🏻